client/components/wizard/navigation-link.jsx

```diff
@@ -11,24 +11,33 @@ import Gridicon from 'gridicons';
  */
 import Button from 'components/button';
 
-const NavigationLink = ( { direction, href, translate } ) => {
-	const text = ( direction === 'back' ) ? translate( 'Back' ) : translate( 'Skip for now' );
+class NavigationLink extends Component {
+	static propTypes = {
+		direction: PropTypes.oneOf( [ 'back', 'forward' ] ).isRequired,
+		href: PropTypes.string,
+		text: PropTypes.string,
+		translate: PropTypes.func.isRequired,
+	}
 
-	return (
-		<Button compact borderless
-			className="wizard__navigation-link"
-			href={ href }>
-			{ direction === 'back' && <Gridicon icon="arrow-left" size={ 18 } /> }
-			{ text }
-			{ direction === 'forward' && <Gridicon icon="arrow-right" size={ 18 } /> }
-		</Button>
-	);
-};
+	getText = () => {
+		const { direction, text, translate } = this.props;
 
-NavigationLink.propTypes = {
-	direction: PropTypes.oneOf( [ 'back', 'forward' ] ).isRequired,
-	href: PropTypes.string,
-	translate: PropTypes.func.isRequired,
-};
+		return direction === 'back' ? text || translate( 'Back' ) : text || translate( 'Skip for now' );
```

Comment:
I'm going to leave the "Skip for now" text for now since I borrowed it directly from the signup flow. It may change when I get a UX review of the WPJM wizard done though.